### ISSUE TITLE
Which set used in Table 2 of paper? 

### ISSUE THREAD

**Issue by muneebaadil** [LimBee/NTIRE2017#14]
I am having a slight confusion regarding the dataset used for Table 2 written in paper.

The figure caption reads: "Performance comparison between architectures on the DIV2K validation set". The DIV2K validation consists of 100 images (800-900)
However, in section 4.4, it is stated that, "Evaluation is conducted on the 10 images of
DIV2K validation set"  

I have a confusion: 
Q. 'DIV2K validation set' refer to which set? The actual validation set consisting of 100 images or your custom validation set you created (10 images) during training? 

Also, kindly mention the file-names of images used in Table2 computation, thanks.

**Comment by limbee** [comment]
We apologize for your confusion. 
Actually, '10 images of' written in section 4.4 is the typo and should be removed from the sentence. All reported DIV2K results in Tables 2 and 3 are based on 100 images (801 ~ 900), not part of them. This will be fixed in the next updated version of the paper.
During training, the number of images used for validation is 10, but of course the images were disjoint from 801~900.
